.github/workflows/ci.yaml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on: [workflow_dispatch, pull_request]
+
+env:
+  SECRET_KEY: thisisasecret
+  DATABASE_URL: postgresql://postgres:password@localhost:5432/postgres
+
+jobs:
+  ci:
+    strategy:
+      matrix:
+        python-version: ["3.6"]
+        platform: [ubuntu-20.04]
```

Comment:
I think we can remove the `strategy` part since explicitly specified the version of python and the platform in this file.

Reply:
resolved in 6086dc7e1b6a6776c961b563c5bb3d461e5cf540